docs/api-reference/io-time-series-change-point.md

```diff
@@ -1,7 +1,10 @@
 ### Input and Output Columns
-There is only one input column and its type is <xref:System.Single>.
-This estimator adds the following output columns:
+There is only one input column and produces a column typed to a 4-element vector.
```

Comment:
What are the constraints on the input column?

Reply:
Separate input and output into two sentences

Reply:
Will do
```
There is only one input column.
The input column must be <xref:System.Single> where a <xref:System.Single> value indicates a value at a timestamp in the time series.
```

---
In reply to: [278258149](https://github.com/dotnet/machinelearning/pull/3546#discussion_r278258149) [](ancestors = 278258149)